packages/api/src/api/project/index.ts

```diff
@@ -2,10 +2,12 @@ import { Router } from 'express';
 import { post } from './post';
 import { mountUserMiddleware } from '../../middleware/mountUserMiddleware';
 import { contributors } from './contributors';
+import { details } from './id';
 
 export const project = Router();
 
 // project.get('', /* don't mount user */, get);
 project.use(mountUserMiddleware);
 project.post('', post);
 project.use('/contributors', contributors);
+project.use('/id', details);
```

Comment:
You need the colon for express to understand that it's a variable instead of just `id` as the path
```suggestion
project.use('/:id', details);
```